lib/stripe/util.rb

```diff
@@ -133,7 +133,8 @@ def self.convert_to_stripe_object_with_params(data, params, opts = {})
       when Hash
         # Try converting to a known object class.  If none available, fall back
         # to generic StripeObject
-        obj = object_classes.fetch(data[:object], StripeObject)
+        object_name = data[:object] || data["object"]
```

Comment:
This way there's 0 performance change for most people, and then anyone working with data that has had the keys `stringify`-ed for whatever reason will still be able to use this method.

Reply:
Does the rest of `convert_to_stripe_object_with_params` work well with unsymbolized objects?

Reply:
Yeah, it works exactly the same since the object construction `construct_from` already symbolizes the whole object just to be safe: https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/stripe_object.rb#L86